docs/tutorials/first_steps/06_create_workflow.rst

```diff
@@ -173,61 +175,58 @@ we can see its contents (2).
 Add a second workflow step
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We will now develop a notebook to count the flights in the filtered data file.
+We will now use a second script to count the flights in the filtered data file.
 As before, we will fast-forward through this step by downloading the solution.
+You can copy over `the interactive notebook
```

Comment:
The sentence about the "interactive notebook" is confusing. I think we can remove it.

Reply:
I agree - I was trying to put in some hint of interactivity but since the code doesn't do anything other than what the scripts are doing it's a bit moot. I've removed it.